packages/ui/src/components/cell/cell.native.tsx

```diff
@@ -26,7 +26,8 @@ interface CellProps extends PressableProps<Theme> {
 export function Root({ caption, icon, title, children, ...props }: CellProps) {
   const theme = useTheme<Theme>();
   const itemLayout = <ItemLayout actionIcon={children} captionLeft={caption} titleLeft={title} />;
-  const defaultColor: string = theme.colors['ink.text-primary'];
+  // This is the default color of all icons, why do we need this?
```

Comment:
I'm not sure about this either. We set the color to `'ink.action-primary-default` in `<Icon` so I think thats sufficient?

Reply:
In removing, I could also remove our use of `cloneElement` here.

Reply:
I think that's good if we can remove `cloneElement` too as in this case it's not necessary